### PR TITLE
Fixes review title

### PR DESCRIPTION
**Pull request by memotoro**
Hello.

Title in the review page tries to be a link to the page, but the text replacement is not working. Also some missing HTML tag was found. Simple change really.

### CODE REVIEW

File: app/views/spree/admin/reviews/index.html.erb [hunk 1]
```diff
@@ -96,7 +96,7 @@
             <% end %>
           </td>
           <td class="align-center">
-            <%= review.user_id ? link_to(review.user.try(:email), [:admin, review.user]) : I18n.t("spree.anonymous") %></p>
+            <%= review.user_id ? link_to(review.user.try(:email), [:admin, review.user]) : I18n.t("spree.anonymous") %><br />
```
Comment (aldesantis) [inline app/views/spree/admin/reviews/index.html.erb:99]: I'd rather have this as a paragraph I think. Does it cause any issues with the spacing?

File: app/views/spree/admin/reviews/edit.html.erb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <% content_for :page_title do %>
-	<%= I18n.t("spree.editing_review_for_html", link: link_to(@review.product.name, product_path(@review.product))) %>
+	<%= I18n.t("spree.editing_review_for_html", link: @review.product.name) %>
```
Comment (aldesantis) [inline app/views/spree/admin/reviews/edit.html.erb:2]: I think the `link` placeholder is a bit misleading here. We either rename it or make the link work. 🙂

Comment (memotoro) [reply app/views/spree/admin/reviews/edit.html.erb]: It is fixed. I renamed it.

File: app/views/spree/admin/reviews/index.html.erb (hunk 1 above)
Comment (memotoro) [reply app/views/spree/admin/reviews/index.html.erb]: Sure, it is fixed now. It was missing the opening paragraph markup.

**Pull request merged**